src/app/core/models/user.ts

```diff
@@ -1,7 +1,7 @@
-export class User {
+export interface User {
     _id?: string;
-    email?: string;
+    email: string;
     firstName?: string;
     lastName?: string;
```

Comment:
I was thinking about all optional properies - they shoudn't be optional - only when they really need to